src/App.jsx

```diff
@@ -1,7 +1,9 @@
+
 import './App.scss';
```

Comment:
according to the style guide, import styles should be at the bottom of the imports list